.example.pypirc

```diff
@@ -0,0 +1,7 @@
+[distutils]
+index-servers = forgejo
+
+[forgejo]
+repository = https://things.com
+username = {username}
+password = {password}
```

Comment:
Not sure what this file is for

Reply:
local pypi server template. will remove if you want.